ubuntu_install_requirements.sh

```diff
@@ -47,6 +48,7 @@ if [[ $PATH != *pyenv* ]]; then
 fi
 
 pyenv install -s 2.7.5
+pyenv install -s 3.8.0
```

Comment:
I don't think there's any Linux distro that at the moment uses Python 3.8 as default
I suggest to use 3.7, also to align with upstream Openstack development

Reply:
@elfosardo Thanks for the review. Your suggestion is addressed.